CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+website/content/docs/contributors/community.md
```

Comment:
we cannot link to site. We have to put a `CONTRIBUTING.md` (symlink fine) under root.